src/network/networkd-dhcp-server.c

```diff
@@ -24,6 +24,11 @@
 #include "string-util.h"
 #include "strv.h"
 
+/* rfc8925 - IPv6-Only Preferred Option for DHCPv4 3.4.
+ * MIN_V6ONLY_WAIT: The lower boundary for V6ONLY_WAIT. Value: 300 seconds
+ */
+#define MIN_V6ONLY_WAIT_USEC (300 * USEC_PER_SEC)
```

Comment:
Duplicated. Please move the `define` in sd-dhcp-lease.c to dhcp-protocol.h or so somewhere appropriate, and include the header.